ecoapp-comments-2/editor.txt

```diff
@@ -0,0 +1,9 @@
+Dave Schimel (Editor):
+
+Your revision should be minor but I'd ask that you reflect on one issue.  You focus a fair amount of attention and analysis on a particular set of plant functional types.  This term has come to mean the limited set of types found in DGVMs and remote sensing based biogeography, though it was originally a more ecological concept, for example in the work of Grimes and colleagues, whose concept was actually trait based.  The PFTs defined and used in models were a convenience in the 1990s that have become enshrined in hard-to-change code, but in light of this study and others make less and less sense.
+
+For example, the PFTs defined by convenience in modeling and limited data, as well as detectability by AVHRR and MODIS, show more detail and more types in the less-diverse parts of the world while lumping the highly diverse tropics into a much smaller array.  Butler et al (2017) as well as radar-based remote sensing show that the assumption that the tropic forests represent a single functional type too be false, with in particular Africa and tropical South America having systematically different values.
+
+While PFT-models have great inertia, work like this paper and others shows even the concept to be less a useful intermediate and more a pragmatic compromise given 20-year old knowledge and not obviously a good way to continue on.  Your own work, and some of the concluding discussion shows that the CLM PFTs and they ilk are not particularly well-founded.  Both reviewers pushed you in the previous revision to be less accepting of PFTs and I also encourage you strongly to add to the discussion a critical assessment in light of correlation and covariance structure of the current or likely future versions of the PFT approach.
+
+You have a unique vantage with these data and I encourage you to be less bound to the status quo.  And, speaking as one of the team that selected the CLM PFTs many years ago, I can tell you that they have more credence than the amount of thought that went into picking them justifies!  This needn't be a major revision, but it is worth some thought.  New emerging data, similar to global remote sensing when the CLM scheme was built, may enable entirely new approaches.
```

Comment:
Given Dave's nudge, and the reading and citing patterns of the average reader, I wonder if we should add a tiny bit in a more prominent position (e.g. a few words toward the end of the abstract about enabling more dynamic/fluid PFT definitions)